release/src/main/scripts/build_release_candidate.sh

```diff
@@ -170,26 +174,50 @@ if [[ $confirmation = "y" ]]; then
   git clone ${GIT_REPO_URL}
   cd ${BEAM_ROOT_DIR}
   git checkout ${RELEASE_BRANCH}
+  git push origin "${RELEASE_BRANCH}"
+  RELEASE_COMMIT=$(git rev-parse --verify HEAD)
 
-  echo '-------------------Generating Python Artifacts-----------------'
-  cd sdks/python
-  virtualenv ${LOCAL_PYTHON_VIRTUALENV}
+  echo '-------------------Creating Python Virtualenv-----------------'
+  python3 -m venv ${LOCAL_PYTHON_VIRTUALENV}
```

Comment:
This changes from a py2 virtualenv to py3 venv. I do not know if that makes a difference. Pointing out in case this is important.

Reply:
Thanks for pointing this out. I wasn't sure which python version I should use, but since 2 is sunsetting I decided to use 3 . Now I am thinking is using fstrings was the good idea since it requires 3.6+.
WDYT @tvalentyn ?

Reply:
Python 3 sounds good. The version of Python 3 would likely be 3.6 or higher giving that most 3.5 is reaching EOL, so not too concerned about fstrings.